### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by john-breton**
As the mediator and UI will be communicating directly for most of the demo, I thought it would help development to have the two merged into one branch to make integration go a bit quicker.

### CODE REVIEW

File: src/main/java/userinterface/UserInterface.java [hunk 1]
```diff
@@ -1,7 +1,476 @@
...
+        				if (ruName.toUpperCase().equals("BACK")) {
+        					break;
+        				}
+        				createCarrierAndRu(ruName, mediator, input);
+        				break;
+        			case "4":
+        				//4.List carriers on RU
+        				System.out.println("Please enter a new Radio Unit name:\n"
+                                + "For example: LTE#1\n"
+                                + "Enter 'Back' to go back to the menu\n");
```
Comment (matthewkwsiu) [inline src/main/java/userinterface/UserInterface.java:81]: Incorrect print statement?

File: src/main/java/userinterface/UserInterface.java [hunk 2]
```diff
@@ -1,7 +1,476 @@
...
+        				ruName = input.next();
+        				if (ruName.toUpperCase().equals("BACK")) {
+        					break;
+        				}
+        				createCarrierOnRu(ruName, mediator, input);
+        				System.out.println("Carrier has been successfully created on " + "Radio Unit " + "'" + ruName + "'");
+        				break;
+        			case "3":
+        				//3.Create Carrier and Radio Unit
+        				System.out.println("Please enter a new Radio Unit name\n"
```
Comment (matthewkwsiu) [inline src/main/java/userinterface/UserInterface.java:68]: Pardon, should this print statement include information on the carrier?

File: src/main/java/userinterface/UserInterface.java [hunk 3]
```diff
@@ -1,7 +1,476 @@
...
+	                            + "Enter 'Back' to go back to the menu\n");
+	    				ruName = input.next();
+	    				if (ruName.toUpperCase().equals("BACK")) {
+	    					break;
+	    				}
+	    				mediator.printVendor(ruName);
+	    				break;
+        			case "7":
+        				//7.List RU alarm status level
+	    				System.out.println("Please enter the name of the Radio Unit you want to list the valarm status level of:\n"
```
Comment (matthewkwsiu) [inline src/main/java/userinterface/UserInterface.java:112]:
```suggestion
	    				System.out.println("Please enter the name of the Radio Unit you want to list the alarm status level of:\n"

```
Small typo

File: src/main/java/userinterface/UserInterface.java [hunk 4]
```diff
@@ -1,7 +1,476 @@
...
+	 * 
+	 * @param input                 Scanner instance to get user input
+	 * @return CarrierFrequencies   Carrier frequency band
+	 */
+	private static CarrierFrequencies chooseFreqBand(String ratType, Scanner input) {
+		CarrierFrequencies freqBand = null;
+		System.out.println("Please choose Carrier Frequency Band:");
+		if (ratType == "LTE") {
+			System.out.println(
+					  "1.	LTE_BAND_1  = 1920 MHz\n"
```
Comment (matthewkwsiu) [inline src/main/java/userinterface/UserInterface.java:388]: I'm not super familiar with the carrier frequency yet, but would you be able replace these with the enum classes in the future (as a TODO item)?

**Review by matthewkwsiu — COMMENTED** [review]
Briefly reviewed the code and made some small print statement comments. I didn't look very deeply into code logic and implementation.

**Comment by john-breton** [comment]
I'm going to make an effort to address most of the comments Matthew has brought up before this gets merged. It'll help to make everything integrate smoothly once I get the mediator setup.

**Pull request merged**